plugins/hm-stack/hm-stack.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Plugin Name: HM Stack Integration
+ * Description: Load the HM Stack API & Endpoint functionality.
+ * Version: 0.1.0
+ * Author: Human Made
+ * Author URI: http://hmn.md
+ * Text Domain: hm-stack
+ *
+ * @package HMStackIntegration
+ */
+
+namespace HM_Stack;
+
+use HM_Stack\Endpoint;
+
+// Load files.
+require_once 'api.php';
+require_once 'endpoint.php';
+
+// Register endpoint controller
+add_action( 'rest_api_init', function() {
+	Endpoint\register_routes();
```

Comment:
This is a bit of a misnomer I think - and Endpoint is typically a route/method pair like "POST /wp/v2/users", so `endpoint->register_routeS` seems kinda backwards. Any reason we didn't just follow the "REST_Controller" naming?

Reply:
I'm a little confused by this, the Controller does [use a method](https://github.com/WordPress/WordPress/blob/master/wp-includes/rest-api/endpoints/class-wp-rest-controller.php#L38) called `register_routes`... 

Are you proposing that we call this `EndpointS\register_routes()`?

Reply:
The hierarchy in the REST API is:

Controller
  - route 1
  - route 2
    - endpoint a
    - endpoint b

Etc. That is - a Controller controls one or more routes, and there's one or more endpoints per route. Therefore, it would make sense to have this as REST_Controller\register_routes

Reply:
Right, I see - that makes perfect sense. Will do 👍